vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit.cc

```diff
@@ -237,7 +237,8 @@ void Reddit::OnUserPage(
   const std::string publisher_name = GetPublisherName(response);
   const std::string user_id = GetUserId(response);
   const std::string publisher_key = GetPublisherKey(user_id);
-  const std::string media_key = GetMediaKey(user_name, REDDIT_MEDIA_TYPE);
+  const std::string media_key =
+      braveledger_media::GetMediaKey(user_name, REDDIT_MEDIA_TYPE);
```

Comment:
namespace not needed as you are in the same namespace

Reply:
Right, but https://github.com/brave/brave-core/pull/2635#discussion_r291574625. When should we not be listing namespace

Reply:
we should try to avoid them when not needed (in the same namespace).

Reply:
removed